test/api/v3/integration/user/POST-user_feed_pet_food.test.js

```diff
@@ -78,4 +101,42 @@ describe('POST /user/feed/:pet/:food', () => {
       expect(body.message).to.eql(res.message);
     });
   });
+
+  context('sending user activity webhooks', () => {
```

Comment:
can this test be moved into the previous `sending user activity webhooks` context block instead of creating a new one?